elide-datastore/elide-datastore-aggregation/src/test/java/com/yahoo/elide/datastores/aggregation/example/CardinalityTestBean.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2019, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.datastores.aggregation.example;
+
+import com.yahoo.elide.annotation.Include;
+import com.yahoo.elide.datastores.aggregation.annotation.Cardinality;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+/**
+ * Test default behavior of {@link Cardinality}.
+ */
+@Entity
+@Include(rootLevel = true)
+@Cardinality
+public class CardinalityTestBean {
```

Comment:
Instead of creating test beans like this, I think we should try to create a real world model that is intuitive.  Let's create a model around PlayerStats for a video game.  All the models ideally should make sense in that context.

Reply:
Ok. Done